index.js

```diff
@@ -3,7 +3,7 @@
 
 var BasePlugin = require('ember-cli-deploy-plugin');
 
-var Promise = require('ember-cli/lib/ext/promise');
+var Promise = require('rsvp').Promise;
```

Comment:
RSVP doesn't seem to be in package.json. Is that going to be a problem?

Reply:
Good point, i have added it